### PR TITLE
New component register esal

### PR DESCRIPTION
**Pull request by Oiranca**
## Proposed changes

Added radio ande checkbox into type.ts from InpuntFieldForm, created molecules with name FormRadio and created two organisms FormRegisterContactPerson and FormRegisterEsal.

## Types of changes

What types of changes does your code introduce to HuellaPositiva?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, 
don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging 
your code._

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: frontend/src/infrastructure/repositories/Convocatory.repository.test.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ describe('convocatoryRepository', () => {
     const exampleSkill: Skill = { name: 'Nombre skill', description: 'description' };
     const proposal: Convocatory = {
       title: 'Example Title',
-      organizer: 'Example Esal',
```
Comment (Oiranca) [inline frontend/src/infrastructure/repositories/Convocatory.repository.test.ts:20]: Se cambió al hacer le refactor name, no es nada importarte porque son los nombres del ejemplo

File: frontend/src/infrastructure/view/components/organisms/Forms/FormRegisterEsal/FormRegisterEsal.tsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                        value={'Lanzarote'}
+                        checked={false}
+                      />
+                      <FormRadio
+                        title={''}
+                        type={'radio'}
+                        name={'La Graciosa'}
+                        value={'La Graciosa'}
+                        checked={false}
+                      />
```
Comment (javidiazdom) [inline frontend/src/infrastructure/view/components/organisms/Forms/FormRegisterEsal/FormRegisterEsal.tsx]: Duplicidad de código innecesaria. ¿No podrías introducir esto en bucle que itere sobre los nombres de las diferentes islas?

**Review by javidiazdom — CHANGES_REQUESTED** [review]
Sería interesante que le echaras un vistazo a la creación de los FormRadios en Form Register Esal. Estoy seguro de que se puede reducir el tamaño del archivo con un bucle.

**Review by javidiazdom — APPROVED** [review]
Correct, this is much better.

**Pull request merged**